hw/top_earlgrey/dv/chip_rom_tests.hjson

```diff
@@ -587,6 +587,93 @@
       ]
       run_timeout_mins: 180
     }
+    {
+      name: rom_e2e_jtag_debug_test_unlocked0
+      uvm_test_seq: chip_sw_rom_e2e_jtag_debug_vseq
+      sw_images: [
+        "//sw/device/silicon_creator/rom/e2e:img_test_unlocked0_exec_disabled:4",
+      ]
+      en_run_modes: ["sw_test_mode_rom"]
+      run_opts: [
+        "+use_jtag_dmi=1",
+        "+en_uart_logger=1",
+        "+use_otp_image=OtpTypeCustom",
+      ]
+      reseed: 1
+      run_timeout_mins: 120
+    }
+    {
+      name: rom_e2e_jtag_debug_dev
+      uvm_test_seq: chip_sw_rom_e2e_jtag_debug_vseq
+      sw_images: [
+        "//sw/device/silicon_creator/rom/e2e:img_dev_exec_disabled:4",
+      ]
+      en_run_modes: ["sw_test_mode_rom"]
+      run_opts: [
+        "+use_jtag_dmi=1",
+        "+en_uart_logger=1",
+        "+use_otp_image=OtpTypeCustom",
+      ]
+      reseed: 1
+      run_timeout_mins: 120
+    }
+    {
+      name: rom_e2e_jtag_debug_rma
+      uvm_test_seq: chip_sw_rom_e2e_jtag_debug_vseq
+      sw_images: [
+        "//sw/device/silicon_creator/rom/e2e:img_rma_exec_disabled:4",
+      ]
+      en_run_modes: ["sw_test_mode_rom"]
+      run_opts: [
+        "+use_jtag_dmi=1",
+        "+en_uart_logger=1",
+        "+use_otp_image=OtpTypeCustom",
+      ]
+      reseed: 1
+      run_timeout_mins: 120
+    }
+    {
+      name: rom_e2e_jtag_inject_test_unlocked0
+      uvm_test_seq: chip_sw_rom_e2e_jtag_inject_vseq
+      sw_images: [
+        "//sw/device/examples/sram_program:sram_program:5",
+      ]
+      en_run_modes: ["sw_test_mode_rom"]
+      run_opts: [
+        "+use_jtag_dmi=1",
+        "+use_otp_image=OtpTypeLcStTestUnlocked0",
```

Comment:
shouldn't we use the Bazel generated OTP images again here so that the same images used for FPGA tests are used in DV? i.e., this would be `OtpTypeCustom` like above

Reply:
ah ok, you can just leave this as in then, we make the change when move all tests to use Bazel generated OTP images.

Reply:
after speaking with @alphan, I don't think this will be the case. @alphan confirmed initially the ROM execution OTP bits will be disabled in early manufacturing, so we probably should have this match the FPGA setup, no?

@alphan @tjaychen @moidx to confirm as well

Reply:
Ok, updated.